tests/system/test_root_metronome.py

```diff
@@ -239,6 +239,30 @@ def test_docker_job():
         assert len(client.get_runs(job_id)) == 1
 
 
+def test_metronome_shutdown_with_no_extra_tasks():
+    """ Test for METRONOME-100 regression
+        When Metronome is restarted it incorrectly started another task for already running job run task.
+    """
+    client = metronome.create_client()
+    job_id = "metronome-shutdown-{}".format(uuid.uuid4().hex)
+    with job(job_no_schedule(job_id)):
+        # run a job before we shutdown Metronome
+        client.run_job(job_id)
+        time.sleep(2)
+        common.assert_one_job_run_with_one_task(client, job_id)
+
+        # restart metronome process
+        # this won't work in multi-master setup if the mesos leader is not the same as metronome leader
+        # we can improve this one there is a good way how to get metronome leader from the system (e.g. info endpoint)
+        metronome_leader = shakedown.master_leader_ip()
```

Comment:
Right now you can mark this test with `@shakedown.masters(1)` to run only on a single-master cluster